security/medusa/l2/kobject_process.c

```diff
@@ -29,7 +29,7 @@ medusa_answer_t process_kobj2kern(struct process_kobject *tk, struct task_struct
 
 		new_user = alloc_uid(tk->uid);
 		if (!new_user)
-			return MED_NO;
+			return MED_DENY;
```

Comment:
should be MED_ALLOW; I don't know, why there was MED_NO. It's a reaction to failed alloc_uid(), so as in many other cases, we should return MED_ALLOW

Reply:
In other comment, you proposed to return MED_ERR, is this somehow different? Should not we have the handling consistent?

Reply:
Yes, sorry. Should be MED_ERR